pkg/controller/studyjobcontroller/studyjob_controller.go

```diff
@@ -477,6 +477,9 @@ func (r *ReconcileStudyJobController) checkStatus(instance *katibv1alpha1.StudyJ
 			log.Printf("Check Goal failed %v", err)
 		}
 	}
+	if instance.Status.SuggestionCount > instance.Spec.RequestCount{
+		nextSuggestionSchedule = false
```

Comment:
Not only set nextSuggestionSchedule false but mark Studyjob status as complete.
And check RequestCount only when it is larger than 0 (users set value explicitly).

Reply:
@YujiOshima

Reply:
Thanks!